baseframe/templates/baseframe/forms-material.html

```diff
@@ -0,0 +1,334 @@
+{% macro rendererrors(field) -%}
+    {%- if field.errors -%}
+      {%- for error in field.errors %}
+        <div><p class="mui-form--error">{{ error }}</p></div>
+      {%- endfor %}
+    {%- endif %}
+{%- endmacro %}
+
+{% macro renderfield(field, css_class="", widget_css_class="", sidetext=none, tabindex=none, autofocus=false, nolabel=false, style='horiz', rows=none) -%}
```

Comment:
This macro may need to be rewritten to remove all the accumulated cruft, but we can do this in a separate PR.